common/core/logicaldisplaymanager.h

```diff
@@ -29,21 +29,55 @@ class LogicalDisplayManager;
 
 class LogicalDisplayManager {
  public:
+  /**
+  * Constructor that sets the physical display for the class.
+  *
+  * @param physical_display a NativeDisplay object representing a physical
+  *                         display
+  */
   LogicalDisplayManager(NativeDisplay* physical_display);
   ~LogicalDisplayManager();
 
+  /**
+  * Initialize new logical displays up to the passed in total count.
+  *
+  * @param total an unsigned 32 bit integer to dictate the amount of logical
```

Comment:
In general I don't think we *need* to document standard data types, unless the acceptable input is some subset of that data type.